include/util/debug.hpp

```diff
@@ -0,0 +1,88 @@
+#ifndef OSRM_ENGINE_GUIDANCE_DEBUG_HPP_
+#define OSRM_ENGINE_GUIDANCE_DEBUG_HPP_
+
```

Comment:
Jup this is a good change. We have lots of other debug code (all the operator<< implementations for various types) that we could move in here.